src/components/TodoList/TodoList.jsx

```diff
@@ -1,4 +1,12 @@
+import { TodoInfo } from '../TodoInfo';
+
 // Add the required props
-export const TodoList = () => (
-  <>TodoList markup</>
+export const TodoList = ({ todos }) => (
```

Comment:
There is no prop types validation. It's a good practice to validate props using PropTypes. This helps to catch bugs early and makes your code more readable.